apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1775,17 +1775,16 @@ block_txs_list_by_hash_invalid_range(_Config) ->
     ok.
 
 naming_system_manage_name(_Config) ->
-
     {ok, PubKey} = rpc(aec_keys, pubkey, []),
-    PubKeyEnc = aec_base58c:encode(account_pubkey, PubKey),
-    Name       = <<"fooo.bar.test">>,
-    NameSalt   = 12345,
-    NameTTL    = 60000,
-    Pointers   = <<"{\"account_pubkey\":\"", PubKeyEnc/binary, "\"}">>,
-    TTL        = 10,
-    NHash      = aens_hash:name_hash(Name),
-    Fee        = 2,
-    MineReward = rpc(aec_governance, block_mine_reward, []),
+    PubKeyEnc   = aec_base58c:encode(account_pubkey, PubKey),
+    Name        = <<"詹姆斯詹姆斯.test"/utf8>>,
```

Comment:
I see this test is using a name with Unicode non-ASCII chars in it - testing all the way through the HTTP APIs. Good.